src/index.ts

```diff
@@ -6,7 +6,8 @@ import {
   DecorationSet,
   Decoration,
 } from '@codemirror/view';
-import { Range, Extension } from '@codemirror/state';
+import { Range, Extension, Text } from '@codemirror/state';
+import { NodeProp, Tree } from '@lezer/common';
```

Comment:
I assume this throws a linter error in dev now, because the dep. isn't explicitly listed.

Explicitly listing it might create an identical object reference challenge though.

Reply:
Yeah it's a challenge with dependencies and transitive ones. What I really want is to use the transitive dependency from @codemirror/language not include it explicitly to avoid reference issues.